libs/utils/utils.js

```diff
@@ -1126,6 +1126,10 @@ export async function loadDeferred(area, blocks, config) {
     import('../features/personalization/preview.js')
       .then(({ default: decoratePreviewMode }) => decoratePreviewMode());
   }
+  if (config?.dynamicNavKey) {
+    const { default: loadDNStatus } = await import('../features/dynamic-navigation/status.js');
```

Comment:
Looks like this file will load on prod even though you're not showing the status there. Would be good to have an env check here

Reply:
Was following mep's pattern and have the env check in the main function, but can move it here.

Reply:
resolved, updated tests, let me know what you think @meganthecoder